CITATION.bib

```diff
@@ -1,10 +1,7 @@
-@article{JuMP.jl-2017,
-  title={JuMP: A modeling language for mathematical optimization},
-  author={Dunning, Iain and Huchette, Joey and Lubin, Miles},
-  journal={SIAM Review},
-  volume={59},
-  number={2},
-  pages={295--320},
-  year={2017},
-  publisher={SIAM}
+@article{Lubin2023,
+    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Benoît Legat and Juan Pablo Vielma},
```

Comment:
```suggestion
    author = {Miles Lubin and Oscar Dowson and Joaquim Dias Garcia and Joey Huchette and Beno{\^i}t Legat and Juan Pablo Vielma},
```

Reply:
See https://tex.stackexchange.com/questions/57743/how-to-write-%c3%a4-and-other-umlauts-and-accented-letters-in-bibliography
Some versions of LaTeX and use of inputenc, etc... may make it possible to make it work with "î" directly but my experience is that it's best to stay within the ASCII realm ^^